spec/index.bs

```diff
@@ -17401,7 +17401,7 @@ The [=texel block memory cost=] of each of these formats is the same as its
         <td colspan=6>If {{GPUFeatureName/"texture-formats-tier1"}} is enabled
         <td><!-- Vulkan -->
         <td>1
-        <td>&ndash; <!-- no render target -->
+        <td>1 (if {{GPUFeatureName/"texture-formats-tier1"}} is enabled)
```

Comment:
nit: these render target costs can be listed without a condition on the feature, because it's just data about them, not a capability. Ditto below

```suggestion
        <td>1
```